doc/source/categorical.rst

```diff
@@ -671,13 +681,14 @@ If the slicing operation returns either a ``DataFrame`` or a column of type
 
 .. ipython:: python
 
-    idx = pd.Index(["h","i","j","k","l","m","n",])
-    cats = pd.Series(["a","b","b","b","c","c","c"], dtype="category", index=idx)
-    values= [1,2,2,2,3,4,5]
-    df = pd.DataFrame({"cats":cats,"values":values}, index=idx)
-    df.iloc[2:4,:]
-    df.iloc[2:4,:].dtypes
-    df.loc["h":"j","cats"]
+    idx = pd.Index(["h", "i", "j", "k", "l", "m", "n"])
+    cats = pd.Series(["a", "b", "b", "b", "c", "c", "c"],
+                     dtype="category", index=idx)
+    values = [1, 2, 2, 2, 3, 4, 5]
+    df = pd.DataFrame({"cats": cats, "values": values}, index=idx)
+    df.iloc[2:4, :]
+    df.iloc[2:4, :].dtypes
+    df.loc["h": "j", "cats"]
```

Comment:
Hmm this doesn’t seem right - no need for spacing after colon, no?

Reply:
ahh, yeah, right!! thanks for the check!! sorry for my carelessness...